p4-16/spec/P4-16-spec.mdk

```diff
@@ -6849,7 +6888,7 @@ control ingress(inout Header_t h, inout Meta_t m,
                 inout standard_metadata_t standard_meta) {
 
     action a() { standard_meta.egress_spec = 0; }
-    action a_with_control_params(bit<9> x) { standard_meta.egress_spec = x; }
```

Comment:
The only reason to change the name of `a_with_control_params` in this example is to make it shorter, so I can reuse most of this example in a new one, and have more page width to work with for adding priority values and comments.